tools/theme/src/Theme.ts

```diff
@@ -321,15 +355,19 @@ export class Theme extends HTMLElement implements ThemeKindProvider {
                         .get(name)
                         ?.get(
                             resolvedValue +
-                                (name === 'theme' ? '' : themeModifier)
+                                (['theme', 'system'].includes(name)
+                                    ? ''
+                                    : systemModifier)
                         )
                 ) {
                     issues.push(
                         `You have set "${name}='${resolvedValue}'" but the associated theme fragment has not been loaded.`
                     );
                 }
             };
-            checkForAttribute('theme', this.theme, this._theme);
+            // `theme` is deprecated in favor of `system` but maintaining `theme` as a deprecated path.
+            checkForAttribute('theme', this.theme, this._system);
+            checkForAttribute('system', this.system, this._system);
```

Comment:
This checks for `theme` and `system` with equal priority, should there be different messaging when a consumer continues to leverage `theme` here?

Reply:
_Right_. So I have added another else condition to check for `theme` and added the below messaging to surface up the deprecation warning.
```js
else if (
    !Theme.themeFragmentsByKind
        .get(name)
        ?.get(
            resolvedValue +
                (name === 'theme' ? '' : themeModifier)
        )
) {
    issues.push(
        `property theme is deprecated. Please use system, instead. You have set "${name}='${resolvedValue}'" but the associated theme fragment has not been loaded.`
    );
}
```